cmd/clustersynchro-manager/app/options/options.go

```diff
@@ -65,6 +68,7 @@ func NewClusterSynchroManagerOptions() (*Options, error) {
 
 	options.Logs = logs.NewOptions()
 	options.Storage = storageoptions.NewStorageOptions()
+	options.WorkerNumber = 1
```

Comment:
The default number of workers is 1 which is not really suitable, we need to set a larger default value, for example 5 or 15.
@cleverhu @wlp1153468871 what do you think?

Reply:
I think it should be greater than 1 such as 3 or 5.